elastic/kibana
# SentinelOne manual host isolation/release in Alert details

## Summary

Add possibility to Isolate/Release SentinelOne host from Alert details flyout. 
Add support for displaying S1 Agent status in UI.
Add an experimental flag to S1 Connector.
Rename S1 connector actions from `Agent` to `Host`
Add a feature flag to security_solution to control enrollment of this feature.
Update parallel script to support all FTR config options
Add `cypress-data-session` plugin to allow better caching of test data (mostly for Dev experience)

Testing instruction:
1. Ensure you have 
2. From root Kibana folder run https://p.elstc.co/paste/URVrCEcR#aG1X9p3BMCRUDY+IzfIg5mGomcTGxwkYO6RGxSIAyWz
3. In Cypress run ```x-pack/plugins/security_solution/public/management/cypress/e2e/sentinelone/isolate.cy.ts```
4. 💚 

<img width="2375" alt="Zrzut ekranu 2023-11-15 o 12 38 27" src="https://github.com/elastic/kibana/assets/5188868/c7ddc20e-9944-452c-b739-fa2d9fbf072b">
<img width="2374" alt="Zrzut ekranu 2023-11-15 o 12 38 32" src="https://github.com/elastic/kibana/assets/5188868/ab3ced14-0a5c-4f40-a92e-844feb849bb4">
<img width="2370" alt="Zrzut ekranu 2023-11-15 o 12 38 38" src="https://github.com/elastic/kibana/assets/5188868/96ccd237-56a6-449e-979d-f4fe8ffbe048">
<img width="2373" alt="Zrzut ekranu 2023-11-15 o 12 38 46" src="https://github.com/elastic/kibana/assets/5188868/924013aa-79ef-405b-ae73-139cf0644ebf">
<img width="2374" alt="Zrzut ekranu 2023-11-15 o 12 39 17" src="https://github.com/elastic/kibana/assets/5188868/e1ff5b05-8b80-40a9-84b1-dd21bf9e059c">
<img width="2374" alt="Zrzut ekranu 2023-11-15 o 12 39 58" src="https://github.com/elastic/kibana/assets/5188868/15fc5d36-970f-47cb-ae2f-f8a19628e6f4">
<img width="2374" alt="Zrzut ekranu 2023-11-15 o 12 40 03" src="https://github.com/elastic/kibana/assets/5188868/5860a0c9-a6e5-43b9-b37d-aa68e4e71f26">
<img width="2373" alt="Zrzut ekranu 2023-11-15 o 12 40 09" src="https://github.com/elastic/kibana/assets/5188868/5e2c5d41-c96a-4c32-8d51-a8408efea8e3">
